### PR TITLE
test: Use jq in a compatible way and for easier diff analysis

### PR DESCRIPTION
**Pull request by dzafman**
I want to merge this right away because a cherry-pick is part of the luminous backport.

### CODE REVIEW

File: qa/standalone/scrub/osd-scrub-repair.sh [hunk 1]
```diff
@@ -31,16 +31,19 @@ fi
...
-        | (. | (.inconsistents[].selected_object_info | .local_mtime ) |= "----Stripped-----" )
-        | (. | (.inconsistents[].selected_object_info | .last_reqid ) |= "----Stripped-----")
-        | (. | (.inconsistents[].selected_object_info | .version ) |= "----Stripped-----" )
-        | (. | (.inconsistents[].selected_object_info | .prior_version ) |= "----Stripped-----" )
-        | ( . | (.inconsistents[].shards[].object_info | .mtime ) |= "----Stripped-----" )
-        | ( . | (.inconsistents[].shards[].object_info | .local_mtime ) |= "----Stripped-----" )
-        | ( . | (.inconsistents[].shards[].object_info | .last_reqid ) |= "----Stripped-----" )
-        | ( .| (.inconsistents[].shards[].object_info | .version ) |= "----Stripped-----" )
-        | ( .| (.inconsistents[].shards[].object_info | .prior_version ) |= "----Stripped-----" )'
+jqfilter='def walk(f):
```
Comment (tchaikov) [inline qa/standalone/scrub/osd-scrub-repair.sh]: very cool!

**Comment by dzafman** [comment]
retest this please

**Comment by tchaikov** [comment]
retest this please.

**Comment by dzafman** [comment]
Scrub test on teuthology passed:

http://pulpito.ceph.com/dzafman-2018-04-17_12:51:42-rados:standalone-wip-zafman-testing-distro-basic-smithi

standalone test passed:
qa/run-standalone.sh osd-scrub-repair.sh 

retest this please
dashboard failure

**Pull request merged**